packages/jira-adapter/test/change_validators/projects/project_assignee_type.test.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ *                      Copyright 2024 Salto Labs Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { toChange, InstanceElement } from '@salto-io/adapter-api'
+import { createEmptyType } from '../../utils'
+import { projectAssigneeTypeValidator } from '../../../src/change_validators/projects/project_assignee_type'
+import { ASSIGNEE_TYPE_FIELD } from '../../../src/constants'
+
+describe('projectAssigneeTypeValidator', () => {
+  const projectType = createEmptyType('Project')
+  let projectInstance: InstanceElement
+
+  beforeEach(() => {
+    projectInstance = new InstanceElement('project', projectType, {})
```

Comment:
nit: if we are using `projectType` only here
```suggestion
projectInstance = new InstanceElement('project', createEmptyType(PROJECT_TYPE), {})
```